test/DynamoCoreTests/LibraryTests.cs

```diff
@@ -32,7 +32,14 @@ public override void Setup()
             libraryCore.Compilers.Add(ProtoCore.Language.kImperative, new ProtoImperative.Compiler(libraryCore));
             libraryCore.ParsingMode = ParseMode.AllowNonAssignment;
 
-            var pathManager = new PathManager(new PathManagerParams());
+            var pathResolver = new TestPathResolver();
+            pathResolver.AddPreloadLibraryPath("DSCoreNodes.dll");
+
+            var pathManager = new PathManager(new PathManagerParams
+            {
+                PathResolver = pathResolver
+            });
+
```

Comment:
Because `LibraryTests` class derives from `UnitTestBase`, there isn't any `GetLibrariesToPreload` call and library preloading needs to be done here.